lab_1_keywords_tfidf/main.py

```diff
@@ -33,6 +42,14 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(tokens, list) or not isinstance(stop_words, list):
+        return None
+    for i in tokens:
+        for j in stop_words:
+            if i == j:
+                while j in tokens:
```

Comment:
loop inside the loop inside the loop - too complicated

Reply:
fixed